handler.lua

```diff
@@ -6,14 +6,15 @@
 
 -- Grab pluginname from module name
 local plugin_name = ({...})[1]:match("^kong%.plugins%.([^%.]+)")
+local random = math.random
```

Comment:
locals are faster; previously it would first search the locals for `math`, after a failure, it would continue to lookup `math` as a global, and find it. Then it would find the value `random` inside `math`.

Now it will only lookup `random` and immediately find it as a local variable.